src/components/tracked-controls.js

```diff
@@ -18,7 +17,13 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    hand: {type: 'string', default: 'right'},
```

Comment:
> I was thinking that `3dof-controls` would underneath rely on `tracked-controls`
If `tracked-controls` is a dependency, then `three-dof-controls` needs to replicate all its data for passthrough (yuck) and it's not clear when it should be used.

If done as a related component, then higher layers like `hand-controls` need to know when to specify `three-dof-controls`, and `three-dof-controls` needs to be able to configure `tracked-controls` to call its `updatePose` instead of doing its normal processing.